app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -63,19 +63,49 @@ class PipelineSampleReport extends React.Component {
     return params.selected_genus != 'None' && params.disable_filters != 1;
   }
 
+  //path to NCBI 
+  gotoNCBI(e) {
+    const taxId = e.target.getAttribute('data-tax-id');
+    const ncbiLink = `https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=${taxId}`;
+    window.open(ncbiLink, '_blank');
+  }
+
+  //download Fasta  
+  downloadFastaUrl(e) {
+    const pipelineId = e.target.getAttribute('data-pipeline-id');
+    const taxLevel = e.target.getAttribute('data-tax-level');
+    const taxId = e.target.getAttribute('data-tax-id');
+    console.log(pipelineId, taxLevel, taxId, 'fasta params');
+    location.href = `/pipeline_outputs/${pipelineId}/fasta/${taxLevel}/${taxId}/NT_or_NR`;
+  }
+
+  displayTags(taxInfo, reportDetails) {
+    return (
+      <span className="link-tag">
+        <i data-tax-id={taxInfo.tax_id} onClick={this.gotoNCBI} className="fa fa-link cloud" aria-hidden="true"></i>
```

Comment:
if taxInfo.tax_id is negative, we shouldn't show the link. The negative taxid is internal for our use, which usually means genus/family doesn't exist